doc/dev/installation.rst

```diff
@@ -423,6 +423,54 @@ To generate html files for the documentation for Catalyst:
 
 The generated files are located in ``doc/_build/html``
 
+Known Issues
+------------
+
+.. tabs::
+
+   .. group-tab:: Linux Debian/Ubuntu
+
+      .. warning::
+
+        If you get this error: 
+
+        .. code-block:: console
+          
+          cannot find -lstdc++: No such file or directory
+
+        you might need to install a recent version of ``libstdc``. E.g.: 
+
+        .. code-block:: console
+          
+          sudo apt install libstdc++-12-dev
```

Comment:
I think it would be useful to have some to link back to where the information came from (e.g. a GitHub issue, slack thread, etc). I guess it doesn't necessarily have to appear in the published docs, but at least as a comment so that we can go back and get more info if needed.

Reply:
> I didn't find a resource for the libstdc issue.

Where did you take the info from?

Reply:
I have found a resource in the issue discussion, thanks!